### PR TITLE
fix: charts not responsive

### PR DESCRIPTION
**Pull request by huai-jie**
close https://github.com/denoland/saaskit/issues/251

Before
image

After
image

### CODE REVIEW

File: static/app.css
Comment (iuioiua) [inline static/app.css:1]: Could this instead be a `<style>` in the `<Head />` for just the stats page?

**Review by iuioiua — CHANGES_REQUESTED** [review]
Can we insert the new class in a `<style>` in the `<Head />` for the stats page only? We should also note somewhere that this is a workaround for [Fresh Charts not being responsive](https://github.com/denoland/fresh_charts/issues/11).

**Comment by iuioiua** [comment]
Sorry, @huai-jie. To clarify, I'm happy to merge this PR, as long as we note that it's a workaround.

**Comment by huai-jie** [comment]
> Can we instead insert the new class in a `<style>` in the `<Head />` for the stats page? We should also note somewhere that this is a workaround for [Fresh Charts not being responsive](https://github.com/denoland/fresh_charts/issues/11). I've made a suggestion for that issue.

done it

**Review by iuioiua — APPROVED** [review]
LGTM! Thank you, @huai-jie!

**Pull request merged**